com.unity.render-pipelines.universal/Editor/Camera/UniversalRenderPipelineCameraUI.Skin.cs

```diff
@@ -0,0 +1,91 @@
+using System;
+using System.Linq;
+using UnityEngine;
+using UnityEngine.Rendering.Universal;
+
+namespace UnityEditor.Rendering.Universal
+{
+    internal static partial class URPCameraUI
```

Comment:
should this one be UniversalRenderPipelineCameraUI ?

Reply:
Thanks :)